draft-foudil-securitytxt.md

```diff
@@ -421,8 +421,8 @@ sign-header      =  <headers and line from section 7 of [RFC4880]>
 
 sign-footer      =  <OpenPGP signature from section 7 of [RFC4880]>
 
-unsigned         =  *line [canonical-field eol *line] [lang-field eol] *line
-unsigned         =/ *line [lang-field eol *line] [canonical-field eol] *line
+unsigned         =  *line [can-field eol] *line contact-field eol *line [lang-field eol] *line
```

Comment:
The "contact-field eol" ought to be in parenthesis to ensure that there is always a line break after the Contact directive.

```suggestion
unsigned         =  *line [can-field eol] *line (contact-field eol) *line [lang-field eol] *line
```

<hr>

I thought I'd quickly also show what disambiguating the line would look like (though I agree a human programmer could probably figure it out regardless):

```abnf
unsigned-begin         =  [*line can-field eol] (*line contact-field eol) [*line lang-field eol] ; order unimportant
unsigned               =  unsigned-begin eol
```

I don't think it's that much more confusing now that I see it's just moving the `*line`s to be inside the brackets. Though this is not too important, as discussed above.

Reply:
done